app/controllers/support/locations_controller.rb

```diff
@@ -15,18 +15,28 @@ def edit
 
     def update
       if site.update(site_params)
-        redirect_to support_provider_locations_path(provider), flash: { success: t("support.flash.updated", resource: "Location") }
+        redirect_to support_provider_locations_path(provider), flash: { success: t("support.flash.updated", resource: flash_resource) }
       else
         render :edit
       end
     end
 
+    def destroy
+      site.destroy!
```

Comment:
I think sites need to be `discarded` otherwise it could cause issues with downstream systems I.e. apply

Reply:
done